workbench-cypress/cypress/e2e/database/releases.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { testDBHeader } from "@utils/dbHeaders";
+import { newExpectation, newShouldArgs } from "@utils/helpers";
+import { runTests } from "@utils/index";
+
+const pageName = "Releases page";
+const dbName = "us-jails";
+const currentPage = `/database/${dbName}/releases`;
+const connectionName = "CypressTestConnection";
+const hasDocs = true;
+
+describe(pageName, () => {
+  const notExist = newShouldArgs("not.exist");
+  const beVisible = newShouldArgs("be.visible");
+
+  const commonTests = [
+    newExpectation(
+      "should not find empty releases message",
+      "[data-cy=release-list-no-releases]",
+      notExist,
+    ),
```

Comment:
I think all of these can use the helper functions like `shouldNotExist`, etc. We probably haven't moved all of these to use those functions in dolthub/hosted yet. Been trying to update them whenever I touch a cypress file